tasks/init.js

```diff
@@ -141,12 +129,7 @@ yargs.getInstance('init')
     'Where run on an exising project existing files will not be altered, delete existing files in order to change ' +
     'properties.',
     '',
-    'Both the npm and bower packages are initially set private which you will need to clear in order to publish.',
-    '',
-    'Available IDE tasks include:',
-    IDE_LIST.join(', '),
-    '',
-    'Any given IDE is initialised per its task defaults. Use the task separately to review these options.'
+    'Both the npm and bower packages are initially set private which you will need to clear in order to publish.'
   ].join('\n')))
```

Comment:
Same observation as the previous PR: Let' look into getting the really long help messages into a separate text file and load them in at run time.